src/client/datascience/data-viewing/dataViewer.ts

```diff
@@ -174,16 +209,19 @@ export class DataViewer extends WebviewPanelHost<IDataViewerMapping> implements
             if (e instanceof JupyterDataRateLimitError) {
                 traceError(e);
                 const actionTitle = localize.DataScience.pythonInteractiveHelpLink();
-                this.applicationShell.showErrorMessage(e.toString(), actionTitle).then((v) => {
-                    // User clicked on the link, open it.
-                    if (v === actionTitle) {
-                        this.applicationShell.openUrl(HelpLinks.JupyterDataRateHelpLink);
-                    }
-                });
+                this.applicationShell.showErrorMessage(e.toString(), actionTitle).then(
+                    (v) => {
+                        // User clicked on the link, open it.
+                        if (v === actionTitle) {
+                            this.applicationShell.openUrl(HelpLinks.JupyterDataRateHelpLink);
+                        }
+                    },
+                    (e) => traceError(e)
+                );
                 this.dispose();
             }
             traceError(e);
-            this.applicationShell.showErrorMessage(e);
+            this.applicationShell.showErrorMessage(e).then(noop, (e) => traceError(e));
```

Comment:
I'd just use `noop` for `showErrorMessage`, its not like we can do anything with these errors.
Its unlike it would fail, & unlikely we'll ever see this & do anything with it..
Nohting to change, just saying that I think its unnecessary.